src/App.js

```diff
@@ -3,28 +3,75 @@ import logo from './logo.svg';
 import './App.css';
 import axios from 'axios';
 import Form from './components/Form';
+
 import Friends from './components/Friends'
+import UpdateFriend from './components/UpdateFriend';
+import {Route }from 'react-router-dom';
+
 class App extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      friends: []
+      friends: [],
+      activeFriend: ''
     }
+    
   }
 
   
-  postFriend = friend => {
-    axios.post('http://localhost:5000/friends', friend)
-    .then(res => console.log(res))
-    .then(res => alert(`You have successfully added to the list!`))
-    .catch(err => console.log(err))
-   }
+
+ postFriend = friend => {
+    axios
+      .post('http://localhost:5000/friends', friend)
+      .then(res => {
+        this.setState({ friends: res.data });
+        this.props.history.push('/');
+      })
+      .catch(err => console.log(err));
+  };
+
+  deleteFriend = (id) => {
+    //console.log('hello')
+    axios
+      .delete(`http://localhost:5000/friends/${id}`)
+      .then(res => {
+       console.log(res);
+       window.location.reload();
+       //this.props.history.push('/')
```

Comment:
You won't need to reload the window to display updates after deleting a friend if you do the following:
• Insert `this.setState({ friends: res.data });` in the `.then` callback
• Pass the `friends` state variable to `<Friends />` instead of letting that component use its own state variable for the friends object array